benchmarks/benchmarks/api/bench_builtin_apply_edges_hetero.py

```diff
@@ -0,0 +1,48 @@
+import time
+import dgl
+import torch
+import numpy as np
+import dgl.function as fn
+
+from .. import utils
+
+
+@utils.benchmark('time', timeout=600)
+@utils.parametrize('num_relations', [5, 50, 500])
+@utils.parametrize('format', ['coo', 'csr'])
+@utils.parametrize('feat_size', [8, 128, 512])
+@utils.parametrize('reduce_type', ['u->e']) #, 'e->u'])
+
+def track_time( num_relations, format, feat_size, reduce_type):
+    device = utils.get_bench_device()
+        dd = {}
+    candidate_edges = [dgl.data.CoraGraphDataset(verbose=False)[0].edges(), dgl.data.PubmedGraphDataset(verbose=False)[
+        0].edges(), dgl.data.CiteseerGraphDataset(verbose=False)[0].edges()]
```

Comment:
we should develop a proper workload for testing the heterogeneous graph API.
the performance improvement or overhead cannot really be seen in this dataset. we should include the benchmark of the knowledge graphs.

Reply:
Will do that in later PRs.